repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -29,30 +30,31 @@ object Project {
 }
 
 final case class ProjectId(value: String) extends MappedTo[String]
-
 object ProjectId {
   implicit lazy val projectIdFormat: Format[ProjectId] = implicitly[Format[String]].inmap(ProjectId.apply, _.value)
 }
-
 final case class ProjectAdditionRequest(name: String)
-
 object ProjectAdditionRequest {
   implicit lazy val projectAdditionFormat: OFormat[ProjectAdditionRequest] = Json.format[ProjectAdditionRequest]
 }
-
 final case class ProjectDeleteRequest(projectId: ProjectId)
-
 object ProjectDeleteRequest {
   implicit lazy val projectDeleteFormat: OFormat[ProjectDeleteRequest] = Json.format[ProjectDeleteRequest]
 }
-
 final case class ProjectUpdateRequest(projectId: ProjectId, name: String, repository: String)
-
 object ProjectUpdateRequest {
   implicit val updateRequestFormat: OFormat[ProjectUpdateRequest] = Json.format[ProjectUpdateRequest]
 }
 
-final case class Version(value: String) extends AnyVal
+final case class Version(name: String, message: String, target: String, commit: Commit)
+object Version {
+  implicit val versionPlayFormat: OFormat[Version] = Json.format[Version]
+}
+
+final case class Commit(id: String, shortId: String, title: String, message: String)
```

Comment:
`shortId` can be easily calculated using full commit `id`. No need for a separate field in a class,